lib/src/model/tile_model.dart

```diff
@@ -12,15 +14,19 @@ class TileLayer {
   TileLayer({required this.name, required this.extent, required this.features});
 }
 
-abstract class TileFeature {
+class TileFeature {
```

Comment:
do we now have some dead code (subclasses) that should be removed?

Reply:
All the subclasses where in `tile_features.dart`, which has been removed.